woocommerce/woocommerce-admin
# Get currency from `CurrencyContext`

Fixes #5750 

Get currency from `CurrencyContext`.

### Screenshots
<img width="203" alt="螢幕快照 2021-03-31 下午5 52 36" src="https://user-images.githubusercontent.com/56378160/113217650-376d1e80-924c-11eb-9ce5-374336001321.png">

<img width="257" alt="螢幕快照 2021-03-31 下午5 53 16" src="https://user-images.githubusercontent.com/56378160/113217673-3fc55980-924c-11eb-9123-45ab5d61bf90.png">


### Detailed test instructions:
1. Go to WooCommerce home and click the store details
2. Set the country to Japan and continue to Business details
3. Set currently selling on another platform
4. See that the current annual revenue dropdown menu currency match the selected country
